docs/calculations.mdx

```diff
@@ -299,6 +302,14 @@ creative_data_transfer_embodied_emissions_gco2pm =
       default_network_embodied_emissions_gco2e_per_kb[network_type]
 ```
 
+### Creative Delivery - Platform
+
+```
+creative_platform_emissions_gco2pm = 0
```

Comment:
Do we have defaults for creative platforms ?

Reply:
Need to add a type to `templates/ad_tech_platform` for `creative_adserver.yaml`

Reply:
what do you mean by defaults for creative platforms? this is platform emissions related to creative delivery. Ad platforms can serve creatives so we don't need a new type.